### PR TITLE
manifest: Update activity and add data-checker

### PR DESCRIPTION
**Pull request by sparshg**
@tchx84

### CODE REVIEW

File: org.sugarlabs.Memorize.json [hunk 1]
```diff
@@ -55,11 +55,11 @@
                     "type": "git",
                     "url": "https://github.com/sugarlabs/gst-plugins-espeak.git",
                     "tag": "v0.6.0",
-                     "commit": "5cace1227427e25638f2f3447a79528e3c5b6db2",
-                     "x-checker-data": {
-                         "type": "git",
-                         "tag-pattern": "^v([\\d.]+)$"
-                     }
```
Comment (tchx84) [inline org.sugarlabs.Memorize.json:62]: Are these fixes or typos?

File: org.sugarlabs.Memorize.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
                     "x-checker-data": {
                         "type": "git",
                         "tag-pattern": "^([\\d.]+)$"
-                   }
-               }
+                    }
+                }
```
Comment (tchx84) [inline org.sugarlabs.Memorize.json:43]: Are these fixes or typos?

Comment (sparshg) [reply org.sugarlabs.Memorize.json:43]: These are formatting fixes.

File: org.sugarlabs.Memorize.json (hunk 1 above)
Comment (sparshg) [reply org.sugarlabs.Memorize.json:62]: These are formatting fixes.

File: org.sugarlabs.Memorize.json (hunk 2 above)
Comment (tchx84) [reply org.sugarlabs.Memorize.json:43]: For future reference, it's better to keep these fixes in a separate commit. But let's go with this now.

**Review by tchx84 — APPROVED** [review]

**Pull request merged**